.github/workflows/master-e2e.yaml

```
@@ -369,6 +368,7 @@ jobs:
       - name: Bootstrap additional nodes (use iPXE) in pool "worker" (total of ${{ inputs.node_number }})
```

I think this comment has to be updated as nodes in worker pool are now installed via ISO.
```suggestion
      - name: Bootstrap additional nodes (use ISO) in pool "worker" (total of ${{ inputs.node_number }})
```

---

Done!